### PR TITLE
Various minor improvements/fixes.

### PR DESCRIPTION
**Pull request by deiferni**
- Move state-related methods to model, from now on proxies should delegate only.
 - Also allow submitting documents for already scheduled proposals.
 - Use meeting MessageFactory for meeting tab translations.
 - Replace committee name in meeting title with location and also show location in meeting listing.
 - Use plone utilities to format dates/times.
 - Consistently use same field order for proposal fields.
 - Fix some translations.
 - Fix some redirects after creating sql-model based objects

### CODE REVIEW

File: opengever/meeting/locales/fr/LC_MESSAGES/opengever.meeting.po [hunk 1]
```diff
@@ -1,7 +1,11 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
+<<<<<<< HEAD
```
Comment (phgross) [inline opengever/meeting/locales/fr/LC_MESSAGES/opengever.meeting.po]: Merge fail ...

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po [hunk 2]
```diff
@@ -488,3 +488,8 @@ msgstr "Eingereicht"
 msgid "tab_matches"
 msgstr "${amount} Treffer"
 
+#. Default: "Remove from schedule"
+#: ./opengever/meeting/model/proposal.py:89
+msgid "un-schedule"
+msgstr "De-Traktandieren"
```
Comment (phgross) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: `Traktandum entfernen`

File: opengever/meeting/model/meeting.py [hunk 3]
```diff
@@ -115,16 +117,27 @@ def update_model(self, data):
...
 
     def get_start_time(self):
-        return self.start_time.strftime('%H:%M') if self.start_time else ''
+        return self._get_time(self.start_time)
 
     def get_end_time(self):
-        return self.end_time.strftime('%H:%M') if self.end_time else ''
+        return self._get_time(self.end_time)
+
+    def _get_time(self, time):
```
Comment (phgross) [inline opengever/meeting/model/meeting.py]: `def localized_time(self, time):` ?

**Comment by phgross** [comment]
Just some minor things ... otherwise :+1:

**Comment by deiferni** [comment]
@phgross updated, also i sneaked in another commit fixing some redirects.

**Pull request merged**